src/sentry/integrations/slack/webhooks/action.py

```diff
@@ -326,6 +401,47 @@ def _handle_group_actions(
 
         original_tags_from_request = slack_request.get_tags()
 
+        use_block_kit = features.has("organizations:slack-block-kit", group.project.organization)
+        if use_block_kit and slack_request.type == "view_submission":
+            # TODO: if we use modals for something other than resolve, this will need to be more specific
+
+            # Masquerade a status action
+            selection = None
+            values = slack_request.data["view"]["state"]["values"]
+            for value in values:
+                for val in values[value]:
+                    selection = values[value][val]["selected_option"]["value"]
+                    if selection:
+                        break
+                if selection:
```

Comment:
@ceorourke doing multiple `if selection break` seems somewhat not ideal, wonder if there is a better way to do this